### DISCUSSION TITLE
Order of payment methods when rendering Bricks.

### DISCUSSION THREAD

**Discussion by josimarcordeiro** [mercadopago/sdk-js#117] [Ideas]
Hi! I have a suggestion.

It would be important for rendering the payment methods to follow the order defined in the configuration. Currently, regardless of the order, the credit card option appears as the first option for the customer.

follow link with illustration

https://freeimage.host/i/sugestao.HCvgEkg

**Comment by meliguicarvalho** [comment]
Hello! Thank you for the suggestion! This seems like a very good way of implementing the payment methods ordering. We will look into it, if there are any news we will update you.
